packages/styles/stories/Components/generic-tag/generic-tag.stories.js

```diff
@@ -0,0 +1,191 @@
+        <div class="fd-generic-tag fd-generic-tag--information is-active" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__icon sap-icon--message-information" role="presentation" aria-hidden="true" aria-label="information"></span>
+            <span class="fd-generic-tag__name">Information</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+        <br><br>
+        <div class="fd-generic-tag fd-generic-tag--warning is-active" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__icon sap-icon--message-warning" role="presentation" aria-hidden="true" aria-label="warning"></span>
+            <span class="fd-generic-tag__name">Warning</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+    </div>
+    <div>
+        <div class="docs-link-container-header"><b>Focus</b></div>
+        <div class="fd-generic-tag is-focus" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__name">Product Cost</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+        <br><br>
+        <div class="fd-generic-tag fd-generic-tag--error is-focus" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__icon sap-icon--message-error" role="presentation" aria-hidden="true" aria-label="error"></span>
+            <span class="fd-generic-tag__name">Error</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+        <br><br>
+        <div class="fd-generic-tag fd-generic-tag--success is-focus" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__icon sap-icon--message-success" role="presentation" aria-hidden="true" aria-label="success"></span>
+            <span class="fd-generic-tag__name">Success</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+        <br><br>
+        <div class="fd-generic-tag fd-generic-tag--information is-focus" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__icon sap-icon--message-information" role="presentation" aria-hidden="true" aria-label="information"></span>
+            <span class="fd-generic-tag__name">Information</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+        <br><br>
+        <div class="fd-generic-tag fd-generic-tag--warning is-focus" role="button" tabindex="0" aria-roledescription="Object Tag">
+            <span class="fd-generic-tag__icon sap-icon--message-warning" role="presentation" aria-hidden="true" aria-label="warning"></span>
+            <span class="fd-generic-tag__name">Warning</span>
+            <span class="fd-generic-tag__value">3.5M EUR</span>
+        </div>
+    </div>
+</div>
```

Comment:
Please add compact state examples

Reply:
There's visually no difference between compact and cozy, it's only the margin top and bottom that are different. My concern was that it might confuse the user. I added some text, hope it make sense.